packages/core/src/stylable-processor.ts

```diff
@@ -300,6 +300,7 @@ export class StylableProcessor {
 
                     break;
                 case 'property':
+                    this.checkRedeclareSymbol(atRule.params, atRule);
```

Comment:
this is checked only for `@property` because although setting and using css var is considered a declaration, it doesn't redeclare a variable if it exist in the scope?

Reply:
right

Reply:
it's only for the @property